Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -87,8 +96,8 @@
         "repositoryURL": "https://github.com/SDWebImage/SDWebImage.git",
         "state": {
           "branch": null,
-          "revision": "d6367439527663d2038ca445a3c3c4e4bac40d60",
-          "version": "5.12.0"
+          "revision": "2e63d0061da449ad0ed130768d05dceb1496de44",
+          "version": "5.12.5"
```

Comment:
Does SDWebImage needs to be updated in this PR?

Reply:
Just noticed that by building `main` I get this change too. Pinging @nbhasin2 just to know if it's expected! 🙏

Reply:
Hi 👋 
This change is fine. Its a minor version bump only that we'll merge to main in this PR.